.github/workflows/docker-image.yml

```
@@ -39,9 +39,17 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Set APP_VERSION env
+        run: echo APP_VERSION=$(echo ${GITHUB_REF} | rev | cut -d'/' -f 1 | rev ) >> ${GITHUB_ENV}
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:18: Quote this to prevent word splitting

(shellcheck)